orca-core/src/main/groovy/com/netflix/spinnaker/orca/listeners/StageTaskPropagationListener.groovy

```diff
@@ -38,7 +38,7 @@ class StageTaskPropagationListener implements StageListener {
   void afterTask(Persister persister, Stage stage, Task task, ExecutionStatus executionStatus, boolean wasSuccessful) {
     task.status = executionStatus
     task.endTime = task.endTime ?: System.currentTimeMillis()
-    log.info("Setting task status to ${task.status} (stageId: ${stage.id}, taskId: ${task.id}) [afterTask]")
+    log.info("Setting task status to ${task.status} (stageId: ${stage.id}, taskId: ${task.id}, taskName: ${task.getName()}) [afterTask]")
```

Comment:
I think there's still something fishy going on with listener ordering and was going to use some additional logging to help out.